flocker_bb/builders/flocker.py

```diff
@@ -661,6 +661,19 @@ def getBuilders(slavenames):
                 nextSlave=idleSlave,
                 ))
 
+    # Storage backend builders
+    builders.extend([
+        BuilderConfig(
+            name='flocker-storage-backend-cinder',
+            slavenames=["rackspace-blockdevice"],
+            category='flocker',
+            factory=makeFactory(
+                b'python2.7',
+                tests=["flocker.node.agents.functional.test_cinder"],
```

Comment:
This should use `--testmodule`.  @wallrj has started on a branch that does things this way for the other OpenStack cloud.  I figure we'll fix this to do the right thing in that branch.